site2/docs/administration-geo.md

```diff
@@ -60,11 +60,9 @@ $ bin/pulsar-admin properties create my-tenant \
 
 To update permissions of an existing tenant, use `update` instead of `create`.
 
-### Creating global namespaces
+### Creating geo-replication namespaces
```

Comment:
The creation of a namespace is really the same as other namespaces. The title here could be `Enabling geo-replication on a namespace`

Reply:
good suggestion.